bin/start-uc-cli-in-docker

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -ex;
+
+container_version="0.1.0"
+container_name="unitycatalog-cli"
+network_name="unitycatalog_network"
+network_details=$(docker network ls --format '{{json .}}' | jq -r --arg NETWORK_NAME "$network_name" 'select(.Name == $NETWORK_NAME) | .Name')
```

Comment:
Can this be simplified to something like `docker network ls -q --filter "name=unitycatalog_network"` ?

Reply:
Good point! i will try. I haven't used the --filter before so I'll give it a shot. Thanks for pointing this out.

Reply:
Fixed in [87c710b](https://github.com/unitycatalog/unitycatalog/pull/116/commits/87c710b0b124f2555d165593aa05446463c39da6)